sql/core/src/main/java/org/apache/spark/sql/sources/v2/SupportsRead.java

```diff
@@ -22,9 +22,10 @@
 import org.apache.spark.sql.util.CaseInsensitiveStringMap;
 
 /**
- * An internal base interface of mix-in interfaces for readable {@link Table}. This adds
- * {@link #newScanBuilder(CaseInsensitiveStringMap)} that is used to create a scan for batch,
- * micro-batch, or continuous processing.
+ * A mix-in interface of {@link Table}, to indicate that it's readable.
+ * <p>
+ * This defines {@link #newScanBuilder(CaseInsensitiveStringMap)} that is used to create a scan
+ * builder for batch, micro-batch, or continuous processing.
```

Comment:
I don't think this documentation change is necessary. Looks like it just reformats and rephrases a little.

Reply:
The reason is, it's not an internal interface anymore. With table capability API, users need to implement this interface explicitly.